Hai Reveal Reworked Introduction (#8789)
endless-sky/endless-sky: data/hai/hai reveal 0 prologue.txt

Reviewer: I don't think these "Meet the Team" missions should be using stopovers.
They don't do anything in particular that makes stopovers useful so the stopovers just make it harder to tell where I need to go when looking at the map; I care where I need to go, no where I need to go after going there.

		set "hr: secret leaks"
		event "hr: friend arrives" 30
		event "hr: friend settled in" 60
		fail

mission "Hai Reveal: Player Knowledge"
	landing
	invisible
	to offer
		has "hr: secret leaks"
		or
			has "First Contact: Hai: offered"
			has "First Contact: Unfettered: offered"
	on offer
		set "hr: knew of hai"
		fail

event "hr: friend arrives"

event "hr: friend settled in"

mission "Hai Reveal: Secret Leaks: FW"
	landing
	invisible
	source
		government "Free Worlds"
		attributes "spaceport"
	to offer
		has "hr: secret leaks"
		not "hr: heard the news"
	on offer
		set "hr: heard the news"
		conversation
			`You are relaxing within the spaceport while your ship refuels after conducting a spot of business. On a video screen is some talk show that you haven't previously encountered, but apparently seems to be quite a popular one out of Tarazed.`
			`	A young and fresh-faced man who looks incredibly ill at ease has just been brought out as a surprise guest by the host Jonathon Camembert.`
			`	"So now in our surprise segment, let me introduce you to Ronald Campbell. He once crewed on one of these big trade ships, but after seeing what he saw, he just had to come forward!"`

> 			`	The host stresses the word "had" so much his voice squeaks, but the audience's attention has certainly been gained.`

Author: ```suggestion
			`	The host stresses the word "had" so much his voice squeaks, but it certainly grabs the audience's attention.`
```

			`	"But first, let's take a look at a clip."`
			branch recognize
				or
					has "First Contact: Hai: offered"
					has "First Contact: Unfettered: offered"
			`	They show a short video featuring alien ships, not Pug, or Quarg, but something else. They are engaged in combat with each other, and fire weapons quite unlike anything produced in human space.`

> 			`	You raise your eyebrows. Actual footage of an unknown alien conflict doesn't seem likely.`

Reviewer 2: I think the goal would actually be to not tell the player what they are thinking, and instead sort-of suggest something.

				goto campbell
			
			label recognize
			`	A clip of alien spaceships in combat with weapons quite unlike anything produced in human space is shown, and you recognize that the ships are Shield Beetles hammering away at each other with their Ion Cannons. This must be a video of the conflict between the Hai and Unfettered.`
			
			label campbell

> 			`	When the clip ends, Campbell is encouraged by Camembert to talk about where he got the footage. Campbell is well presented, although his suit is definitely rented - you couldn't imagine how a crewman would be able to afford a suit like that on their salary - and he's clearly anxious. You can make out the beads of sweat on his brow as he considers his answer, and his neat suit is slightly crumpled in the way that happens when people wipe sweat off their palms a lot. When Campbell speaks, it is with a paranoid manner that is plain to see even through a screen. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war; a war waged by an alien species known as the "Hai." Although he does not give their exact whereabouts, he says they can be reached, and in turn reach us, with just a hyperdrive.`

Reviewer 2: ```suggestion
			`	When the clip ends, Campbell is encouraged by Camembert to talk about where he got the footage. Campbell is well presented, although his suit is obviously not his own - you couldn't imagine how a crewman would be able to afford an outfit like that on their salary - and he's clearly anxious. You can make out the beads of sweat on his brow as he considers his answer, and his neat suit is slightly crumpled in the way that happens when people wipe sweat off their palms a lot. When Campbell speaks, it is with a paranoid manner that is plain to see even through a screen. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war; a war waged by an alien species known as the "Hai." Although he does not give their exact whereabouts, he says they can be reached, and in turn reach us, with just a hyperdrive.`
```

> 			`	When the clip ends, Campbell is encouraged by Camembert to talk about where he got the footage. Campbell is well presented, although his suit is definitely rented - you couldn't imagine how a crewman would be able to afford a suit like that on their salary - and he's clearly anxious. You can make out the beads of sweat on his brow as he considers his answer, and his neat suit is slightly crumpled in the way that happens when people wipe sweat off their palms a lot. When Campbell speaks, it is with a paranoid manner that is plain to see even through a screen. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war; a war waged by an alien species known as the "Hai." Although he does not give their exact whereabouts, he says they can be reached, and in turn reach us, with just a hyperdrive.`

Reviewer 3: I never noticed there was a rhyme and I wrote that line. Maybe I should read over my work more next time : )

				decline
			label hai
			`	The existence of the Hai is intended to be a secret to most humans. News like this being widespread could easily cause a crisis if many people believe it and find the Hai themselves...`
				decline

mission "Hai Reveal: Secret Leaks: R/S"
	landing
	invisible
	source
		government "Republic" "Syndicate"
		attributes "spaceport"
	to offer
		has "hr: secret leaks"
		not "hr: heard the news"
	on offer
		set "hr: heard the news"
		conversation
			`You are relaxing at the spaceport while your ship refuels, when a breaking news broadcast comes on; it's not urgent enough to actually interrupt scheduled content, but urgent enough to fill the first ad break.`
			`	The severe-looking anchorwoman bores down the camera with an expression that demands attention.`
			`	"An ex-crewman employed by an itinerant trader has come forward to media about hostile aliens just beyond our borders. He claims to have brought video proof of an ongoing conflict that threatens to spill over and drag all of humanity back to war. Stay tuned for our full segment at six."`

> 			`	You raise your eyebrows. Actual footage of an unknown alien conflict doesn't seem likely. That such news should come out so soon after the Pug smacks of a hoax. A quick glance at the clock shows you only have ten minutes to wait, so you stick around for the full segment out of curiosity.`

Author: I don't see it as any different than the bombing news event before the start of the main campaign. There you're not given a choice on whether or not you ask someone why there's a large crowd gathered around a video screen. The point of both of these missions is to act as exposition for the player with regards to the upcoming story.

More generally, there are plenty of places in the game where we have the player character do something without the player's input. While we do want to give the player a choice as to how their character acts in certain scenarios, it wouldn't be reasonable to say that in every scenario where a player could make a choice that we should give them a choice.

At the end of the day, we're trying to craft stories, and we should probably avoid allowing the player to make choices in missions that would ruin that story. Say you don't stick around. Then what? You just don't get this information? Or if you gain this information in a different manner within this same conversation, then what was the point of the choice?

> 			`	You raise your eyebrows. Actual footage of an unknown alien conflict doesn't seem likely. That such news should come out so soon after the Pug smacks of a hoax. A quick glance at the clock shows you only have ten minutes to wait, so you stick around for the full segment out of curiosity.`

Reviewer 3: This is fair. I still have some questions about that (and about the Martini and Geminus bombings conversation another day) but I'll leave it as it is.

			`	It is only after this that the anchorwoman reveals that, while this was initially treated as a hoax, one of their own reporters posed a timely question to a well-respected trader and caused quite the stir when he slipped up and confirmed the existence of the aliens known as "Hai."`
				goto table
			
			label seen2
			`	You keep going about your business, but eventually turn a corner and come face-to-face with a life-sized holographic display of a human news anchor with the familiar background and logo of a human news organization. This is being projected up from a portable device set up on a table that several Hai are gathered around. You recognize it as the same news report you saw being aired outside of the Free Worlds, with the anchorwoman reporting on Campbell's appearance on the Tarazed talk show.`
			
			label table
			`	Two of the Hai at the table seem to have been seeing this for the first time. They make several loud comments that feel like they are some mixture of shock, excitement, and distress, though you only have their tone to go off.`

> 			`	No wonder the place is abuzz though, this is clearly a major event for the Hai.`

Reviewer 2: ```suggestion
			`	No wonder the place is abuzz; this is clearly a major event for the Hai.`
```

				goto hoax
			
			label seen3
			`	"Here, let me show you what they've got going around," he says as he pulls out a Hai data pad. He hits play on a video and shows you the same clip you saw circulating in human space.`
			
			label hoax
			`	"Turns out," he says, "this was almost treated as a hoax, but some journo caught one of the regular traders off-guard and he let it slip. So now people know, or at least think they know, about the Hai."`
			choice
				`	"How could this happen?"`
				`	"What'll happen now?"`
					goto now
			`	The guy laughs. "It's not as if the Hai actually enforce their secrecy," he says, heavily emphasizing the word "enforce."`
			`	"The merchants might fire a crewman for trying to leak information, but if the conditions are right to make more money by leaking it, someone was bound to eventually." He shrugs. "Guess the conditions were finally right for it."`
			label now

> 			`	He looks around for a moment with an almost conspiratorial expression. "My money is on some kind of crisis happening though. Keeping the secret is worth a lot of money to certain people. It's going to be an interesting few months I think."`

Reviewer 4: I try to err on the side of fewer commas, but I agree with bot of bene-dictator's suggestions here. A comma in front of `though` and another after `months`.

	`	"Meet my ship, the <npc>, in orbit after I'm finished with my business here." You assure Alondo that you'll be there, and the Hai escort him away.`
		accept
	
	label busy
	action
		event "hr: join later" 20
	`	"I'll probably reach out to you for help later, then. In a situation like this, the Hai need humans they can trust. They seem to trust me enough so far, so hopefully they'll also trust you by proxy, or at the very least you can gain their trust in short order.`
	`	"Well, good luck with whatever it is that you're busy with," he says. You say your goodbyes to Alondo, and the Hai escort him away.`
		decline

mission "Hai Reveal: Early Arrival: Alondo"
	name "Unlikely Encounter"
	description "Head to <destination> with Alondo to learn more about the fallout of the existence of the Hai being revealed to billions in human space."
	source
		government "Hai"
		not attributes "uninhabited" "station"
	destination "Hai-home"
	to offer
		not "hr: met alondo"
		has "First Contact: Hai: offered"
		has "hr: heard the news"
		has "event: hr: friend arrives"
		not "event: hr: friend settled in"
		has "event: joined the free worlds"
	on offer
		conversation "hr: meet alondo"

> 	npc save

Reviewer 5: For consistency with other NPCs that use both, this should be `npc accompany save` rather than `npc save accompany`

> 	npc save

Reviewer 6: Done for both

		government "Hai"
		personality escort timid
		ship "Centipede" "Emissary"
	on complete
		set "hr: meet the team"

mission "Hai Reveal: Late Arrival: Alondo"
	landing
	name "Unlikely Encounter"
	description "Head to <destination> with Alondo to learn more about the fallout of the existence of the Hai being revealed to billions in human space."
	source
		government "Hai"
		not attributes "uninhabited" "station"
	destination "Hai-home"
	to offer
		not "hr: met alondo"
		has "First Contact: Hai: offered"
		has "hr: heard the news"
		has "event: hr: friend settled in"
		has "event: joined the free worlds"
	on offer
		set "hr: met alondo"
		conversation "hr: meet alondo"
	on decline
		set "First Contact: Hai: offered"
	on accept
		set "First Contact: Hai: offered"

> 	npc save

Reviewer 5: Other missions have `npc accompany save` rather than `npc save accompany`.

			label briefing
			`	Alondo leads you to a meeting room that feels large for the number of people present: it looks like a lecture hall that could easily fit a hundred people, but there are only six here. Aside from Alondo and yourself, there is a shorter man in what you recognize as a Republic diplomatic uniform messing with a holoprojector that doesn't appear to be working, a taller man in a suit sitting next to a woman in a formal dress in the front row, and an older looking Hai woman sitting alone at the end of a row of seats.`
			`	"Glad you finally decided to show up, Alondo," says the man in the front.`
			`	"And you brought a guest," says the Hai.`
			`	"I did," Alondo responds. "This is Captain <first> <last>. Member of the Free Worlds Council during the war. Helped repel the Pug invasion. A capable pilot. Am I missing anything?"`
			choice
				`	"That sounds like everything."`
				`	"Just that I'm good looking."`
			`	"There we go!" says the man who was fiddling with the holoprojector as it whirs to life. "No time for introductions. We're late enough as is. Everyone, please sit down."`

> 			`	The holoprojector displays a series of graphs. You're no expert in data analysis, but a highlighted "chance of total crisis" at the bottom ranging from 61% to 96% does not look promising.`

Reviewer 2: Just wondering, how are they generating this percentage?

Author: It's made up, just like half of all statistics are.

Reviewer 2: I mean how are the Hai getting this?

Author: This is from Xilin, not the Hai.

Reviewer 3: I imagine that the Hai government has a department of very dedicated statisticians whose sole purpose is to calculate the exact numbers of the chance they'll be public knowledge in human space...?

... does it particularly matter?

Reviewer: > It's made up, just like half of all statistics are.

Did you make that up?

Reviewer 2: > This is from Xilin, not the Hai.

OK, I didn't see the context.

Reviewer 2: > I imagine that the Hai government has a department of very dedicated statisticians whose sole purpose is to calculate the exact numbers of the chance they'll be public knowledge in human space...?

Sure!

> ... does it particularly matter?

Probably not.

			`	"It has been noted that the character of these new arrivals is also different. Where once a majority of new arrivals were lone explorers or lost captains, they are now increasingly captains following breadcrumbs while looking for the Hai, or even captains bringing large groups of 'tourists' with the promise of witnessing the aliens from the news reports. Further research will need to be done to determine the threat that these new arrivals pose to further escalating the situation."`
			choice
				`	"What's the problem with new arrivals?"`
				`	"Do we not have a way of blocking captains from entering the wormhole?"`
					goto block

> 			`	"We do not have the means of handling a large influx of humans," says the Hai. "For decades, we have dealt with a small trickle of humans arriving through the wormhole. We have grown acustomed to it. But if that trickle were to suddenly become a downpour, then our services would quickly become overrun."`

Reviewer 2: When I got to Hai space, I didn't notice any services. Is there something I'm missing?

Reviewer 6: You were presumably not disembarking from a ship with the intent of settling there.

Author: Just any general services that you might expect of a city or a spaceport to have in supporting a population, whether that be permanent or temporary. If you have tens or even hundreds of thousands of extra people that you weren't expecting to have to deal with, even if they're just tourists who are passing through, then refueling, sanitation, food services, housing, etc. will all be strained and could break.

					goto humble
			`	"You're every inch what I expected," he assures you.`
				goto mission
			label humble
			`	For a moment his expression falters, and then he seems almost relieved. "You may be a better person than I expected to meet.`
			label mission
			`	"Now, about that task. I mentioned during our briefing that we need to determine how much of a threat these new arrivals might be. Not a threat in the sense that they will become violent, but in the sense that they could exacerbate the situation, such as by returning to our space and making a big stink about what they've found here. The best place to go for this would be <stopovers>."`
			`	"I would like to follow and observe," says Sayari.`
			`	"You're free to come," says Xilin. "Let me pack a few items into your ship, Captain <last>, and we can be on our way."`
				accept
	on stopover
		conversation
			`It doesn't take long after landing on <planet stopovers> to find new arrivals. Every so often a human ship will land and its crew will disembark while gawking at the spaceport, making it very obvious that they're newcomers.`

> 			`	Xilin records information about each captain, including their pilot's license, ship registration, how exactly they found their way to the Hai, and more. One of the more concerning events is when a Star Queen full of passengers lands in the spaceport.`

Reviewer 5: 1 ship per day would be at most about 30,000 humans in a year, and probably closer to 3,000 (most ships are towards the smaller end), not all of whom will be there to stay. That wouldn't be that significant given that the population of Hai space is on the order of tens of billions.

> 			`	Xilin records information about each captain, including their pilot's license, ship registration, how exactly they found their way to the Hai, and more. One of the more concerning events is when a Star Queen full of passengers lands in the spaceport.`

Reviewer 5: Could have used fuel pods instead of ramscoops, they have the advantages of being cheaper, smaller (if you already had 400 fuel and only need one more jump), and not requiring extra time sitting around near the star to refuel.

> 			`	Xilin records information about each captain, including their pilot's license, ship registration, how exactly they found their way to the Hai, and more. One of the more concerning events is when a Star Queen full of passengers lands in the spaceport.`

Reviewer 6: > 1 ship per day would be at most about 30,000 humans in a year, and probably closer to 3,000 (most ships are towards the smaller end), not all of whom will be there to stay. That wouldn't be that significant given that the population of Hai space is on the order of tens of billions. Even assuming the average ship coming through the wormhole carries about 100 people, all of whom want to stay in Hai space, humans would still only be less than a tenth of a percent of the population after 300 years.
> ...
> With the same assumptions, an average of two ships in an hour would mean about a 1% increase in the total population of Hai space over 60 years, which could be sufficient to maybe be problematic if the Hai have lost the ability to handle significant population growth as a result of their millennia of stagnation.

Arguably the issue is not the total Hai population, but the population of Allhome specifically, since the availability of services to receive people is concentrated there where basically ever human ever lands for the first time. Also, unexpected immigration tends to cause strain at like 0.005-0.01%...

> I do prefer the phrasing of your version though, so maybe keep the other changes, but change `in the past day` to `in the last hour`, or maybe `in the last hour alone`? 

I would rather double the numbers per day rather than change the interval.

> 			`	Xilin records information about each captain, including their pilot's license, ship registration, how exactly they found their way to the Hai, and more. One of the more concerning events is when a Star Queen full of passengers lands in the spaceport.`

Reviewer 7: Considering we're seeing two more come in just in the time of this conversation, I think having two in the past day before that is fine. The acceleration of doubling that is already right there in the dialogue.

> 			`	Xilin records information about each captain, including their pilot's license, ship registration, how exactly they found their way to the Hai, and more. One of the more concerning events is when a Star Queen full of passengers lands in the spaceport.`

Author: Went a different route.

			`	"You have to be kidding me," Xilin says under his breath while approaching the Star Queen.`
			`	"Welcome, one and all, to the land of the Hai," the ship's captain says while walking down the boarding ramp with all her passengers behind her.`
			`	"Excuse me, Captain. Do you care to answer a few questions?" Xilin shouts up to the captain.`
			`	The captain looks concerned to see Xilin standing there in a Republic uniform, but agrees to talk after shooing her passengers off into the spaceport.`
			`	"What is it you need to know, officer?" the captain asks. Although Xilin isn't a Navy officer, he doesn't correct her.`

> 			`	"I'd like to know why it is that you decide to bring all these passengers to Hai space who had never been here before."`

Author: ```suggestion
			`	"I'd like to know why it is that you decide to bring all these passengers who had never been to Hai space before."`
```

			`	You approach the Berserker as it finishes landing. The captain of the ship exits looking far calmer than the condition of the Berserker would suggest its captain should be. "Uh, hi," he says akwardly. "Can I help you guys?"`
			choice
				`	"What happened to your ship?"`
				`	"Did you get attacked by pirates on the way here."`
			`	The captain looks back at the poor state of his interceptor. "Oh, that. Yeah, I ran into a couple pirates on the way here. Nothing I haven't dealt with before, though."`
			`	"So you've been to Hai space before?" Xilin asks.`
			`	"Sure. Plenty of times," he responds.`
			`	Xilin looks over the interceptor with a quizzical expression. "May I see your pilot's license?"`
			`	"Yeah, no problem," the captain says, as he quickly pulls a license out of his pocket that looks almost as beat up as the interceptor. Xilin jots down the details and hands the license back to the captain. "Thank you. And maybe tag along with a merchant fleet on your way back home."`
			`	"That sounds like a good idea," the captain says before walking off into the spaceport.`

> 			`	You continue to question more captains in the spaceport over the course of the day with Xilin and Sayari. Some already knew of the Hai before, but many are new arrivals who heard of the way to the Hai through whispers in spaceport bars, or were sold coordinates by knowledgeable captains. Everyone seems peaceful enough in interacting with the Hai, but clearly the word is getting out more as time goes on.`

Reviewer 6: In line with not quite so many daily arrivals, adjusting the acquisition of information - and also planting someone that could arguably be counted as an ongoing informant for Xilin to handwave acquiring updated data in the future.
```suggestion
			`	You continue to question more captains in the spaceport over the course of the day with Xilin and Sayari. Most turn out to be repeat visitors or those who have taken to conducting their business entirely in Hai space, but they are all familiar with arrivals who heard of the way to the Hai through whispers in spaceport bars, or were sold coordinates by knowledgeable captains and many are able to provide information about these encounters. One captain in particular has been camped out here just intercepting new captains as they arrive and giving them the induction tour, and has a copious set of notes that he gladly shares with Xilin. It seems that everyone has been peaceful enough in interacting with the Hai so far, but clearly the word is getting out more as time goes on.`
```

Author: Ditto.

> 			`	You continue to question more captains in the spaceport over the course of the day with Xilin and Sayari. Some already knew of the Hai before, but many are new arrivals who heard of the way to the Hai through whispers in spaceport bars, or were sold coordinates by knowledgeable captains. Everyone seems peaceful enough in interacting with the Hai, but clearly the word is getting out more as time goes on.`

Reviewer 4: ```suggestion
			`	You continue to question more captains in the spaceport over the course of the day with Xilin and Sayari. Some already knew of the Hai before, but many are new arrivals who heard of the way through the wormhole through whispers in spaceport bars, or were sold coordinates by knowledgeable captains. Everyone seems peaceful enough in interacting with the Hai, but clearly the word is getting out more as time goes on.`
```

	on visit
		dialog phrase "generic missing stopover or passengers"
	on complete
		conversation
			`You return to the briefing room at the secure port with Xilin and Sayari, and Xilin begins frantically copying notes into a terminal. He then pauses while staring at something on the screen.`

> 			`	"I'm running each of the pilot's licenses and ship registrations through my copy of the Republic's pilot database. They've all come up as valid and with mostly clean records. All except for one." He turns the terminal screen toward you and Sayari, displaying an inputted license ID that the system says does not exist.`

Reviewer 6: In theory he could have done most of this work on a portable device (since he'd have to have a copy of the database not stored on the Hai's system anyway) so this frantic action doesn't make a lot of sense.

Author: > I'd presume this terminal is the location of that database, instead of walking around with such sensitive information

This is what I was thinking. I also imagine this terminal as also being air-gapped with no wireless connections, so it isn't something he could sync with. Note too that at this point in the story, all communications across the wormhole need to be done manually, as there's no sort of connector running through the wormhole to allow information to travel wirelessly, so any digital information that Xilin wants to access he had to physically bring with him.

Outside of the story, though, I just needed some way of having this conversation take place during the on complete, instead of loading everything into the stopover conversation and having nothing left to talk about once we got here.

I'll change the first line to say that he transfers the information into the terminal instead of frantically copying it.

			`	"This is from that Berserker pilot. What's more is that if I run the ID of the ship, it's registered as stolen."`
			`	Sayari gives a look of concern. "Are you suggesting that this captain was a criminal?"`
			`	"Highly likely," Xilin responds. "A pirate with a stolen ship and a counterfeit license."`

> 			`	"I will inform our spaceport authorities to look out for this captain and his ship," Sayari says. "It may also be beneficial if you were to make this database available to us, so that we may catch other potential criminal humans."`

Reviewer 6: I doubt the Hai prosecute humans for crimes not in Hai space. For individuals below a certain threat level, and without an existing arrangement with some Hai entity, it's likely they just monitor them rather than taking any action.

Author: ```suggestion
			`	"Highly likely," Xilin responds. "A pirate with a stolen ship and a counterfeit license."`
			`	"I will inform our spaceport authorities to look out for this captain and his ship," Sayari says. "It may also be beneficial if you were to make this database available to us, so that we may monitor other potential criminal humans."`
```

			`	"Unfortunately, I'd have to gain authorization for that," says Xilin. "But I'll see what I can do."`
			choice
				`	"Should I help track down that captain?"`
					goto track
				`	"So are pirates trying to sneak their way into Hai territory?"`
			`	"That may be the case," Xilin says. "We'd be stupid to suspect this as only having been an isolated incident."`
				goto pirates
			label track
			`	"No, leave that to the Hai," Xilin says. "At least for so long as this is our only lead on pirate activity in Hai territory, although I imagine that we'll find more if we go looking for it."`
			label pirates
			`	"We have had run-ins with your pirates before," Sayari says. "Most of the time they cause only mischief. Very rarely do that raid our fleets and cause large trouble. I fear though that this situation will only embolden them."`
			`	Just then, Alondo poke his head into the room looking somewhat exasperated. "Captain <last>. Could you give Remington a hand? He's out on the landing pads." He poke his head back out of the room, and you hear his heavy footsteps grow quieter as he walks away.`

> 			`	"You go help Remington," Xilin says. "I'll continue my work here."`

Reviewer 6: This transition could be a little less abrupt.

Author: Do you have any suggestion for how to change it?

Looking for the FW campaign, conversations ending like this, with a line from some character, isn't too out of the ordinary. It's been said before that this is a game, not a novel, and in that sense I don't think we necessarily need to have some sort of transition out of every conversation. When I was writing these, I certainly couldn't think of something that wouldn't just be a repetition of something I'd said before. (In a similar vein, I think that the very first line of conversations/mission chains can be a pain to write, because it's almost always some variation of "you land" or "you're in the spaceport when..." so long as we hold that we need some transition into the conversation instead of being able to be somewhat abrupt.)

mission "Hai Reveal: Meet the Team: Samuel"
	name "High Tension"
	description "Remington has heard news of tensions between new arrivals and local Hai on <stopovers>. Return to <destination> when finished."
	source "Hai-home"
	stopover "Cloudfire"
	passengers 2
	blocked "You will need <capacity> in order to take this mission."
	to offer
		has "hr: meet the team"
		has "Hai Reveal: Meet the Team: Xilin: done"
	on offer
		log "People" "Terry Adrianopoulos" `Terry Adrianopoulos is the secretary of Samuel Remington, or at least that's how he puts it. Remington mentioned an internal political situation within the Syndicate while glancing at Terry, so there may be something larger at play with her presence. Hopefully it doesn't conflict with resolving the situation with the Hai.`
		conversation

> 			`You exit the build onto the landing pads with Sayari following behind you and find the tall man and the woman from the front row of the briefing waiting by your ship.`

Reviewer 6: Minor fix - but realistically, there's probably a terminal section for this building which would make more sense - otherwise they'd be competing with not-insignificant wind on a high-rise landing pad.
```suggestion
			`You exit the building onto the landing pads with Sayari following behind you and find the tall man and the woman from the front row of the briefing waiting by your ship.`
```

			branch met
				has "met remington"
			action
				log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public relations. He is acting as the Syndicate's representative in helping to deal with the situation regarding the Hai.`

> 			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department. You may have not heard of me before, but I'm sure you've heard of my work. I..." He stops himself. "You know what, nevermind. We're not here to make small talk. We have bigger things to worry about. I'm looking forward to working alongside you to handle this emerging crisis."`

Reviewer 6: ```suggestion
			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department. You may not have heard of me before, but I'm sure you've heard of my work. I..." He stops himself. "You know what, nevermind. We're not here to make small talk. We have bigger things to worry about. I'm looking forward to working alongside you to handle this emerging crisis."`
```

				goto terry
			label met
			action
				log "People" "Samuel Remington" `Remington is being sent as the representative of the Syndicate to help deal with the situation regarding the Hai.`
			`	"Captain <last>, is it? It's been a good while since we last met," he says while shaking your hand, perhaps a little too firmly. "We met back on Hephaestus near the start of the war. It's been a while, so I'll forgive you if you've forgotten me. I'm looking forward to working alongside you to handle this emerging crisis."`
			label terry

> 			`	You note that his appearance is rather casual compared to what you would expect of someone of importance in the Syndicate, wearing khakis and a collared tee as opposed to the typical suit and tie business attire that you see on Syndicate businessmen. It is certainly in contrast to a young woman standing behind him, who is wearing the usual business attire.`

Reviewer 6: ```suggestion
			`	You note that his appearance is rather casual compared to what you would expect from someone of importance in the Syndicate, wearing khakis and a collared tee as opposed to the typical suit and tie business attire that you see on Syndicate businessmen. It is certainly in contrast to the young woman standing behind him, who is wearing the usual business attire.`
```

			`	"Now, allow me to introduce you to Terry," Remington says while gesturing to the woman behind him.`
			`	"Terry Adrianopoulos. Pleasure to meet you," she says with a broad smile and a slight bow. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She seems at once pleasant and approachable while at the same time being slightly off-putting and uncanny. It's an eerie feeling.`
			`	"Terry is... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him.`
			choice
				`	"Alondo said you needed my help with something."`
					goto help
				`	"The Syndicate sent their PR team to deal with this?"`

> 			`	"I'll have you know I'm considered something of a miracle worker among the Syndicate," Remington says in a sarcastic tone. "The Syndicate was sad to rent me out, but there's also an internal political situation going on within the Syndicate that has been causing me quite the headache recently." His eyes momentarily flick to Terry as he says that, but Remington presses on immediately. "It's nothing I can discuss in detail, with all the NDAs involved, you know, but something I'll certainly enjoy taking a break from."`

Reviewer 4: ```suggestion
			`	"I'll have you know I'm considered something of a miracle worker among the Syndicate," Remington says in a wry tone. "The Syndicate was sad to rent me out, but there's also an internal political situation going on that has been causing me quite the headache recently." His eyes momentarily flick to Terry as he says that, but Remington presses on immediately. "It's nothing I can discuss in detail, with all the NDAs involved, you know, but something I'll certainly enjoy taking a break from."`
```
I'm guessing he's not actually being sarcastic here? I read it as he really does think of himself as a miracle worker but is self-aware enough to realise how arrogant that makes him, so he is acknowledging both facts.

Also, I don't think we need to repeat 'the Syndicate' since the context and the fact he says 'internal' should make it obvious.

			`	"Alondo said you needed my help," you say.`
			label help
			`	"Ah, yes. Well you see, I'd like to travel to <stopovers> to get the lay of the land around the conflict between the two Hai groups. Care to give me a ride there? It'd be just the two of us."`

> 			`	"I will be accompanying you," Sayari says.`

Reviewer 6: This would actually be an excellent point to flag the existence of Hiyamaharu.
I've been considering him to be the senior member of the Hai diplomatic core in specifically the Syndicate Relations division. Remington would likely have met/worked with him before and be a little surprised that Sayari was personally overseeing his business here.

Author: I'll keep this in mind, but I'd like to save that for a future PR, likely one which also removes Sayari from the Wanderer campaign so that she's not in two places at once.

			`	"Just the three of us, then," Remington adds. "The Wah Ki system is where Mr. Campbell's video that's been plastered all over the news was taken. I imagine any new arrivals attempting to look into the validity of the video themselves would find their way there, so it'd be worth checking in on the place."`
				accept
	on stopover
		conversation

> 			`It doesn't take long to prove Remington's hunch correct. You find a group of new arrivals gathered by a group of Hai captains and crewmen, asking them question after question. "Why are you guys hiding up here? Where can I buy some of those weapons of yours? Do you eat acorns?" Some of the Hai look intrigued by the attention they're being given, but other look annoyed.`

Reviewer 6: ```suggestion
			`It doesn't take long to prove Remington's hunch correct. You find a group of new arrivals gathered by a group of Hai captains and crewmen, asking them question after question. "Why are you guys hiding up here? Where can I buy some of those weapons of yours? Do you eat acorns?" Some of the Hai look intrigued by the attention they're being given, but others look annoyed.`
```

> 			`It doesn't take long to prove Remington's hunch correct. You find a group of new arrivals gathered by a group of Hai captains and crewmen, asking them question after question. "Why are you guys hiding up here? Where can I buy some of those weapons of yours? Do you eat acorns?" Some of the Hai look intrigued by the attention they're being given, but other look annoyed.`

Reviewer 4: ```suggestion
			`It doesn't take long to prove Remington's hunch correct. You find a group of new arrivals gathered by a group of Hai captains and crewmen, asking them question after question. "Why are you guys hiding up here?" "Where can I buy some of those weapons of yours?" "Do you eat acorns?" Some of the Hai look intrigued by the attention they're being given, but others look annoyed.`
```

			`	One of the human captains seems to be riling the others up and being rather hostile with his comments. "Do you really think you could invade us? You're like half my size, I could take you on in a fight any day."`
			`	Remington shakes his head. "I'm going to step in before this turns ugly," he says while pulling a small device out of his pocket. It's not anything that you recognize, but it has a Hai appearance to it.`
			`	He shouts to gain everyone's attention before tossing the device between the two groups. As it lands, a life-sized image of a Pug appears, looking almost realistic aside from the fact that its facing angle seems to follow your perspective. Most of the human captains jump back.`
			`	"Can anyone tell me what that is?" Remington asks.`
			`	"That's one of those Pug things!" says one of the human captains.`
			`	"Correct. And I take it you don't like them."`
			`	"Why would I like someone who invaded my home planet?" says another captain.`
			`	"Exactly. Now what's the issue that you take with these Hai here?"`

> 			`	"Well I'm not looking to get my home planet invaded again," the captain responds.`

Author: I'll keep it in mind, but passing over this for now since removing this mention would require changing lines further down.

			`	"Captain <last>," Remington says while turning to you. "Can you tell me what the difference between the Hai and the Pug is?"`
			choice
				`	"The Hai aren't invading us."`
				`	"The Hai are cuter."`
					goto cute
				`	"The Hai are friendly."`
					goto friendly
			`	"But they could!" one of the captains pipes up.`
			`	"Really?" Remington responds. "So because these Hai here are fighting pirates of their own species, you think they could invade us? Could the Syndicate's security forces invade the Republic because they fight off pirates, or the Republic decide to invade the Free Worlds because they fight off pirates? Maybe the answer is yes, but there's a big difference between could and will.`
				goto sayari
			label cute
			`	"I've seen cute dogs turn fearsome before!" one of the captains pipes up.`

> 			`	Remington gives the captain (and you) a slight look of disbelief, but he rolls with it. "Ok, so maybe cute things can be fearsome, but has a fearsome dog ever invaded your home plate?`

Reviewer 6: ```suggestion
			`	Remington gives the captain (and you) a slight look of disbelief, but he rolls with it. "Ok, so maybe cute things can be fearsome, but has a fearsome dog ever invaded your home planet?`
```

			`	Remington speaks with the crowd of humans for a little while longer, eventually convincing them to leave the Hai alone. "Let's head back to <planet>," Remington says. "I've seen enough here. And we should probably think about having the spaceport authority here restrict who can land, at least until all this blows over."`
			`	"Perhaps," Sayari responds.`
	on visit
		dialog phrase "generic missing stopover or passengers"
	on complete
		conversation
			`You return to the briefing room at the secure port with Remington and Sayari. "This Hai holographic tech is pretty nifty isn't it?" Remington says while playing around with the device he used on <planet stopovers>. "Way more advanced than anything we've got back home. It's also pretty useful that there's a port for one of our cables, otherwise I never would have figured out how to get that Pug image on there."`
			`	"Ah, yes," Sayari says. "Many decades of interaction with humans has led us to accommodate much of our technology to your needs. Humans are our guests here, so we saw it only right to make you feel more comfortable. A shame to see a few bad apples spoil the bunch."`
			`	"You have apples up here?" Remington asks.`

> 			`	Sayari chuckles. "Just as we have adapted out technology to human needs, we have also imported many human foods, and learned human sayings. It has all been very exciting."`

Reviewer 4: ```suggestion
			`	Sayari chuckles. "Just as we have adapted out technology to human needs, we have also imported many human foods, and learned human sayings. It has all been very exciting."`
```
```suggestion
			`	Sayari chuckles. "Just as we have adapted our technology to human needs, we have also imported many human foods and learned human sayings. It has all been very exciting."`
```

			`	Sayari pauses for a moment. "Our civilization is tens of thousands of years old. There is little in that time that we have not done on our own. But seeing such a young species has led to much excitement. I wish for that excitement to not turn into strife. This is what I explained to the Hai on <planet stopovers>. 'These humans are young. Be patient with them.'"`
			choice
				`	"Glad to see that you still have a positive view of us."`
				`	"Well we're here to do our best to make that wish come true."`

> 			`	Sayari goes on about the history between Hai and humanity, how a colony exploration team sanctioned by the Republic found the wormhole in Ultima Thule and made first contact between the Hai and Republic. Early on, they used the Quarg as translators between the two parties, but eventually the Hai began to learn the human language, and even some humans began to learn Hai, although it seems much more difficult for a human to speak Hai than the other way around.`

Reviewer 3: No, I'd think there would certainly be more than one language. HR did have a lot of mentions of "Galactic Standard" once but I had removed all those mentions since it was strange that nobody else in the rest of the game ever mentions it.

It could be changed to this:
```suggestion
			`	Sayari goes on about the history between Hai and humanity, how a colony exploration team sanctioned by the Republic found the wormhole in Ultima Thule and made first contact between the Hai and Republic. Early on, they used the Quarg as translators between the two parties, but eventually the Hai began to learn human languages, and even some humans began to learn Hai, although it seems much more difficult for a human to speak Hai than the other way around.`
```

> 			`	Sayari goes on about the history between Hai and humanity, how a colony exploration team sanctioned by the Republic found the wormhole in Ultima Thule and made first contact between the Hai and Republic. Early on, they used the Quarg as translators between the two parties, but eventually the Hai began to learn the human language, and even some humans began to learn Hai, although it seems much more difficult for a human to speak Hai than the other way around.`

Reviewer 2: Is there a standard language, or just human translation devices everywhere?

> 			`	Sayari goes on about the history between Hai and humanity, how a colony exploration team sanctioned by the Republic found the wormhole in Ultima Thule and made first contact between the Hai and Republic. Early on, they used the Quarg as translators between the two parties, but eventually the Hai began to learn the human language, and even some humans began to learn Hai, although it seems much more difficult for a human to speak Hai than the other way around.`

Reviewer 3: I think there would be auxiliary languages, but this is also 3000AD humanity. Second is probably very likely.

> 			`	Sayari goes on about the history between Hai and humanity, how a colony exploration team sanctioned by the Republic found the wormhole in Ultima Thule and made first contact between the Hai and Republic. Early on, they used the Quarg as translators between the two parties, but eventually the Hai began to learn the human language, and even some humans began to learn Hai, although it seems much more difficult for a human to speak Hai than the other way around.`

Reviewer 7: I always got the impression there was one distinct "primary" human language, with others hanging on for sub/cultural relevance. From the Hai perspective, especially in a diplomatic context, referring to this primary language as "the human language" would indeed be appropriate.

The Hai probably still have at least a few of their own old cultural languages hanging around, after all, even if just in academic settings.

> 			`	While she is telling stories, Terry walks in and motions to you. "<last>. I need someone with a ship and enough skill to not get blown up. Meet me by the landing pads when you're ready."`

Reviewer 6: Again, a little abrupt. Perhaps a choice in here to listen (and be dragged away by Terry), or to step away (and be intercepted by Terry)?

Author: Ditto to the other one you said was abrupt.

mission "Hai Reveal: Meet the Team: Terry"
	name "Scout the North"
	description "Scout <waypoints> to determine the frequency of fleets traveling around the wormhole. Return to <destination> when finished."
	source "Hai-home"
	waypoint "Cardax"
	waypoint "Hassaleh"
	waypoint "Moktar"
	waypoint "Oblate"
	waypoint "Rajak"
	waypoint "Rigel"
	waypoint "Sumar"
	waypoint "Ultima Thule"
	waypoint "Volax"
	to offer
		has "Hai Reveal: Meet the Team: Alondo: done"
		has "Hai Reveal: Meet the Team: Xilin: done"
		has "Hai Reveal: Meet the Team: Samuel: done"
	on offer
		conversation

> 			`When you exit onto the landing pads, you find that Terry has just finished refueling your ship for you. "Alright, Captain. I have a simple mission for you. We need someone to scout out the ship frequencies in various uninhabited systems in the Far North. We know how many merchants are arriving here, but we're not sure how many are attempting and failing, for one reason or another. I need you to fly through these systems and report back here with what you find."`

Reviewer 6: ```suggestion
			`When you exit onto the landing pads, you find that Terry has just finished ensuring your ship is refuelled for you. "Alright, Captain. I have a simple mission for you. We need someone to scout out the ship frequencies in various uninhabited systems in the Far North. We know how many merchants are arriving here, but we're not sure how many are attempting and failing, for one reason or another. I need you to fly through these systems and report back here with what you find."`
```

			`	She takes a deep breath before saying: "<waypoints>. Think you can remember all those? Good." She's didn't even wait for a response. "Stick around in each system if you can, as I want to know just how many merchants, pirates, and Navy ships are passing through that region."`
			`	She tosses you a data pad containing coordinates to all the systems she mentioned and starts walking back inside. "Just come back here once you're finished."`
				accept
	on visit
		dialog phrase "generic waypoint on visit"

mission "Hai Reveal: Meet the Team: Sayari"
	landing
	name "Important Message"
	description "Deliver an important message from the Hai to the Republic on <stopovers>. Return to <destination> when finished."
	source "Hai-home"
	stopover "Earth"
	to offer
		has "Hai Reveal: Meet the Team: Terry: done"
	on offer
		conversation

> 			`Terry quickly takes a copy of your ship's system logs upon your return. "Thank you, Cpatain. I'll have these analyzed in no time." Terry runs back into the building, passing Sayari on her way out, carrying a small box.`

Reviewer 3: ```suggestion
			`Terry quickly takes a copy of your ship's system logs upon your return. "Thank you, Captain. I'll have these analyzed in no time." Terry runs back into the building, passing Sayari on her way out, carrying a small box.`
```
About `passing Sayari on her way out`, it would look better if it were `Terry runs back into the building carrying a small box, passing Sayari on her way out` but I could see that it might be this way to make her actions seem rushed.

Reviewer 3: Oh I should have known that. thanks. I'll edit it.

> 			`	"Captain <last>. I know we are asking much of you, but I have a task of my own." She hands you the box, which is officially branded by the Hai government. Inside is an encrypted data chip and a metallic badge. There is a symbol minutely engraved into the front that must be the Hai government's emblem. Surrounding it in a circle is some sort of pattern, which reminds you of the codes found on identification cards and licenses in human space. On the other side, there is extensive writing in a script that looks to be Hai. From this, you presume that the badge is some sort of diplomatic ID.`

Reviewer 4: ```suggestion
			`	"Captain <last>. I know we are asking much of you, but I have a task of my own." She hands you the box, which displays official Hai government branding. Inside is an encrypted data chip and a metallic badge, with a symbol minutely engraved into the front that must be the Hai government's emblem. Surrounding it in a circle is some sort of pattern, which reminds you of the codes found on identification cards and licenses in human space. On the other side, there is extensive writing in a script that looks to be Hai. From this, you presume that the badge is some sort of diplomatic ID.`
```

			`	"Please deliver this message to the Republic. I trust that they will disseminate this message to your other governments. This badge should make it easier if they give you trouble."`
			choice
				`	"Consider it done."`
					accept
				`	"Various? We only have two governments."`
			`	"Oh," she says with a smirk, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`
				accept
	on stopover
		conversation

> 			`On approach to the spaceport on Earth, you request to speak to a diplomatic official. Given your renown from the Pug war, you're promise you a "speedy two to three business days" wait. But after showing the diplomatic ID that Sayari gave you, that quickly shrinks to "two hours."`

Reviewer 6: ```suggestion
			`On approach to the spaceport on Earth, you request to speak to a diplomatic official. Given your renown from the Pug war, they promise you a "speedy two to three business days" wait. But after showing the diplomatic ID that Sayari gave you, that quickly shrinks to "two hours."`
```

			`	You meet with Jamal Rahman, the Chief Minister of Foreign Affairs and Xilin's boss. "I take it you've involved yourself with this 'Hai' situation," he says. You nod. "Well I apologize, but I do not have much time for small talk. What is it you need me for?" You hand him the encrypted message that Sayari gave you and explain where it's from. He thanks you, and leaves with the message.`

> 			`	Some hours later, Rahman returns with the an encrypted data chip. "This is our response. Safe travels, Captain."`

Reviewer 4: ```suggestion
			`	Some hours later, Rahman returns with an encrypted data chip. "This is our response. Safe travels, Captain."`
```

	on visit
		dialog phrase "generic stopover on visit"
	on complete
		event "hr: exploration period" 45
		conversation

> 			`You meet Sayari at the secure port and hand her the response from the Republic. "Thank you, Captain <last>," she says. "Not only for this task, but for the other tasks you have completed for us. You've helped us gather much needed information to determine the path forward."`

Reviewer 6: Technically all that information could be acquired without you - but your involvement meant that the team did not have to choose between trusting a Hai captain, or a random merchant captain, and could rely on someone well-vouched for.
```suggestion
			`You meet Sayari at the secure port and hand her the response from the Republic. "Thank you, Captain <last>," she says. "Not only for this task, but for the other tasks you have completed for us. You've helped with many tasks which might otherwise have been sources of tension without you."`
```

			choice
				`	"Don't mention it. I don't do it for the praise."`
				`	"Is there anything else you need me for?"`
					goto more
			`	Sayari smiles. "How humble. Although I suppose you're wondering what do to next.`
			label more

> 			`	"For now, there are no immediate tasks for you. The others will begin developing a plan to execute on, and we will call upon you once you are needed. I suggest you explore our systems while you wait, and gain a better understanding of the Hai people. It will do us well for you to have this knowledge."`

Reviewer 6: ```suggestion
			`	"For now, there are no immediate tasks for you. The others will continue working on the plan to manage the situation, and we will call upon you once you are needed. I suggest you explore our systems while you wait, and gain a better understanding of the Hai people. It will do us well for you to have this knowledge."`
```

# Copyright (c) 2020 MasterOfGrey
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Hai Reveal: Secret Leaks"
	landing
	invisible
	to offer
		has "main plot completed date"
		"days since start" - "main plot completed date" > 230
	on offer
		set "hr: secret leaks"
		event "hr: friend arrives" 30
		event "hr: friend settled in" 60

> 		fail
> 
> mission "Hai Reveal: Player Knowledge"
> 	landing
> 	invisible
> 	to offer
> 		has "hr: secret leaks"
> 		or
> 			has "First Contact: Hai: offered"
> 			has "First Contact: Unfettered: offered"
> 	on offer
> 		set "hr: knew of hai"

Author: ```suggestion
		"hr: knew of hai" = "First Contact: Hai: offered" + "First Contact: Unfettered: offered"
```

			`	"Now though, after the civil war and the Pug invasion, with your new government in the south, someone has taken advantage of the opportunity. Here, let me show you."`
			`	The Hai pulls out a device and gestures for you to sit on a nearby bench.`
			choice
				`	(Sit.)`
					goto sit
				`	(Politely decline.)`
					goto skip
			label clip
			`	The Hai pulls out a device and gestures for you to sit on a nearby bench.`
			label sit
			branch seen1
				has "Hai Reveal: Secret Leaks: FW: offered"
			`	The Hai pulls up a video of some talk show that you haven't previously encountered, but which seems to be a quite popular one out of Tarazed. In it, a young and fresh-faced man is brought out as a surprise guest by the host, Jonathon Camembert.`
			`	"So now in our surprise segment, let me introduce you to Ronald Campbell. He once crewed on one of these big trade ships, but after seeing what he saw, he just had to come forward!"`

> 			`	The host stresses the word "had" so much his voice squeaks, but the audience's attention has certainly been gained.`

Author: ```suggestion
			`	The host stresses the word "had" so much his voice squeaks, but it certainly grabs the audience's attention.`
```

			`	"But first, let's take a look at a clip."`
			`	A clip of alien spaceships in combat with weapons quite unlike anything produced in human space is shown, and you recognize that the ships are Shield Beetles hammering away at each other with their Ion Cannons. This must be a video of the conflict between the Hai and Unfettered.`

> 			`	When the clip ends, Campbell is encouraged by Camembert to talk about where he got the footage. Campbell is well presented, although his suit is definitely rented - you couldn't imagine how a crewman would be able to afford a suit like that on their salary - and he's clearly anxious. You can make out the beads of sweat on his brow as he considers his answer, and his neat suit is slightly crumpled in the way that happens when people wipe sweat off their palms a lot. When Campbell speaks, it is with a paranoid manner that is plain to see even through a screen. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war; a war waged by an alien species known as the "Hai." Although he does not give their exact whereabouts, he says they can be reached, and in turn reach humanity, with just a hyperdrive.`

Author: ```suggestion
			`	When the clip ends, Campbell is encouraged by Camembert to talk about where he got the footage. Campbell is well presented, although his suit is obviously not his own - you couldn't imagine how a crewman would be able to afford an outfit like that on their salary - and he's clearly anxious. You can make out the beads of sweat on his brow as he considers his answer, and his neat suit is slightly crumpled in the way that happens when people wipe sweat off their palms a lot. When Campbell speaks, it is with a paranoid manner that is plain to see even through a screen. He talks at length about an ongoing conflict that threatens to spill over and drag all of humanity back to war; a war waged by an alien species known as the "Hai." Although he does not give their exact whereabouts, he says they can be reached, and in turn reach humanity, with just a hyperdrive.`
```

	`	"Meet my ship, the <npc>, in orbit after I'm finished with my business here." You assure Alondo that you'll be there, and the Hai escort him away.`
		accept
	
	label busy
	action
		event "hr: join later" 20
	`	"I'll probably reach out to you for help later, then. In a situation like this, the Hai need humans they can trust. They seem to trust me enough so far, so hopefully they'll also trust you by proxy, or, at the very least, you can gain their trust in short order.`
	`	"Well, good luck with whatever it is that you're busy with," he says. You say your goodbyes to Alondo, and the Hai escort him away.`
		decline

mission "Hai Reveal: Early Arrival: Alondo"
	name "Unlikely Encounter"
	description "Head to <destination> with Alondo to learn more about the fallout of the existence of the Hai being revealed to billions in human space."
	source
		government "Hai"
		not attributes "uninhabited" "station"
	destination "Hai-home"
	to offer
		not "hr: met alondo"
		has "First Contact: Hai: offered"
		has "hr: heard the news"
		has "event: hr: friend arrives"
		not "event: hr: friend settled in"
		has "event: joined the free worlds"

> 	on offer

Author: No. It's set during the conversation.

		conversation "hr: meet alondo"
	npc accompany save
		government "Hai"
		personality escort timid
		ship "Centipede" "Emissary"
	on complete
		set "hr: meet the team"

mission "Hai Reveal: Late Arrival: Alondo"
	landing
	name "Unlikely Encounter"
	description "Head to <destination> with Alondo to learn more about the fallout of the existence of the Hai being revealed to billions in human space."
	source
		government "Hai"
		not attributes "uninhabited" "station"
	destination "Hai-home"
	to offer
		not "hr: met alondo"
		has "First Contact: Hai: offered"
		has "hr: heard the news"
		has "event: hr: friend settled in"
		has "event: joined the free worlds"
	on offer
		set "hr: met alondo"
		conversation "hr: meet alondo"
	on decline
		set "First Contact: Hai: offered"
	on accept

> 		set "First Contact: Hai: offered"

Author: Leftovers.

	to offer
		has "hr: meet the team"
		has "Hai Reveal: Meet the Team: Alondo: done"
	on offer
		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and recent peace.`
		conversation
			`Sayari follows you out to the landing pads to meet the presenter. "Captain <last>," he says while grasping your hand in a firm handshake. "Xilin Yang, Republic Deputy Minister of Foreign Affairs. I apologize for not introducing myself sooner. It's an honor to meet a hero of the Pug crisis such as yourself."`
			choice
				`	"It was hard-fought, but it was necessary."`
				`	"Thanks, but it wasn't as dramatic as they made it out to be. I'm just happy to have helped where I could."`
					goto humble
			`	"You're every inch what I expected," he assures you.`
				goto mission
			label humble
			`	For a moment his expression falters, and then he seems almost relieved. "You may be a better person than I expected to meet.`
			label mission

> 			`	"Now, about that task. I mentioned during our briefing that we need to determine how much of a threat these new arrivals might be. Not a threat in the sense that they will become violent, but in the sense that they could exacerbate the situation, such as by returning to our space and making a big stink about what they've found here. The best place to go for this would be <stopovers>. I have a sentry there keeping watch and recording new arrivals, but I'd like to meet some of them for myself."`

Author: ```suggestion
			`	"Now, about that task. I mentioned during our briefing that we need to determine how much of a threat these new arrivals might be. Not a threat in the sense that they will become violent, but in the sense that they could exacerbate the situation, such as by returning to our space and making a big stink about what they've found here. The best place to go for this would be <stopovers>."`
```

			`	"I would like to follow and observe," says Sayari.`
			`	"You're free to come," says Xilin. "Let me pack a few items into your ship, Captain <last>, and we can be on our way."`
				accept
	on stopover
		conversation

> 			`You meet with the sentry that Xilin mentioned back on <planet>, who turns out to be a Republic Intelligence officer who's been recording pilot's licenses and ship registration of all new arrivals. He hands over a data stick of notes to Xilin. "They make themselves very obvious with the way they gawk," he says. "We had two new ships come in yesterday, a Freighter and an Osprey, but there's been nothing today as of yet. It's normally only one or two smaller ships a day, if that."`

Reviewer 6: So the thing about there being a Sentry:
1. If they're an established entity there's nothing to stop Xilin simply calling them, for the majority of updates - or having someone else in his employ do a collection run. He could also do a simple collection trip himself at a pinch, because he doesn't need a captain who can facilitate an uncertain schedule of investigating and such.
2. If there's an actual republic agent, they're presumably competent enough (and in less physical danger) to be confronting new captains arriving, which completely negates the value of Xilin wandering around himself.
3. The presence of a republic agent publicly situated in Hai space would _have_ to have acquired some sort of Hai approval, possibly some sort of Hai oversight, and likely some sort shared reporting arrangement with the Hai. This is the nature of intergovernmental relations involving the public deployment of government agents - which means that Sayari would have no need to accompany them to find out anything because she'd have had to have been in the loop for setting it up in the first place.
4. (3.b) If the agent acting as Sentry was a secret agent, then this could work, but then they cannot be confronting pilots publicly, and then Xilin would absolutely not want Sayari to be coming along.

This was why the original suggestion had you _discover_ someone who has every right to be there already - who is simply conscientiously willing to help. A citizen informer doesn't have any of the baggage that an agent does, and cannot be leverage by the Hai government to extract anything from the Republic.

Note: This is not to say that Xilin does/would _not_ have an agent around, he might. A hypothetical agent might even have passed on information about the captain doing the recording so that Xilin could conveniently "find" him.
A public and formally employed person though, creates a number of holes that would need patching.

Author: Then how about just taking the captain that was mentioned at the end and moving them to the front here?
```suggestion
			`	"I would like to follow and observe," says Sayari.`
			`	"You're free to come," says Xilin. "Let me pack a few items into your ship, Captain <last>, and we can be on our way."`
				accept
	on stopover
		conversation
			`The rate at which new ships show up isn't tremendous, so Xilin starts by speak with some of the older captains who have been here prior to the news. "You can learn a lot by sharing a drink with the right merchant," he says to you.`
			`	One captain in particular has been camped out here to intercept new captains as they arrive and give them the induction tour. He also has a copious set of notes that he gladly shares with Xilin. "After I heard the news, I figured I'd help by keeping notes on new arrivals. They make themselves very obvious with the way they gawk," he says. "We had two new ships come in yesterday, a Freighter and an Osprey, but there's been nothing today as of yet. It's normally only one or two smaller ships a day, if that."`
```

> 			`	Xilin decides that it'd be best to search the spaceport for captains that showed up recently, but just before heading off a Star Queen full of passengers lands in the spaceport with everyone on board pressing their faces against the windows.`

Author: ```suggestion
			`	Xilin decides to next search the spaceport for captains that showed up recently, but just before heading off, a Star Queen full of passengers lands in the spaceport with everyone on board pressing their faces against the windows.`
```

			`	"You have to be kidding me," Xilin says under his breath while approaching ship.`
			`	"Welcome, one and all, to the land of the Hai," the ship's captain says while walking down the boarding ramp with all her passengers behind her.`

> 			`	"Excuse me, Captain. Do you care to answer a few questions?" Xilin shouts up to the captain.`
> 			`	The captain looks concerned to see Xilin standing there in a Republic uniform, but agrees to talk after shooing her passengers off into the spaceport.`

Author: ```suggestion
			`	"Excuse me, Captain. Do you care to answer a few questions?" Xilin shouts up to the captain. She looks concerned to see Xilin standing there in a Republic uniform, but agrees to talk after shooing her passengers off into the spaceport.`
```

			`	"So you've been to Hai space before?" Xilin asks.`
			`	"Sure. Plenty of times," he responds.`
			`	Xilin looks over the interceptor with a quizzical expression. "May I see your pilot's license?"`
			`	"Yeah, no problem," the captain says, as he quickly pulls a license out of his pocket that looks almost as beat up as the interceptor. Xilin jots down the details and hands the license back to the captain. "Thank you. And maybe tag along with a merchant fleet on your way back home."`
			`	"That sounds like a good idea," the captain says before walking off into the spaceport.`
			`	You continue to question more captains in the spaceport over the course of the day with Xilin and Sayari. Some already knew of the Hai before, but many are new arrivals who heard of the way to the Hai through whispers in spaceport bars, or were sold coordinates by knowledgeable captains. Everyone seems peaceful enough in interacting with the Hai, but clearly the word is getting out more as time goes on.`
	on visit
		dialog phrase "generic missing stopover or passengers"
	on complete
		conversation

> 			`You return to the briefing room at the secure port with Xilin and Sayari, and Xilin begins frantically copying notes into a terminal. He then pauses while staring at something on the screen.`

Author: ```suggestion
			`You return to the briefing room at the secure port with Xilin and Sayari, and Xilin begins transferring his notes into a terminal. He then pauses while staring at something on the screen.`
```

			label pirates
			`	"We have had run-ins with your pirates before," Sayari says. "Most of the time they cause only mischief. Very rarely do they raid our fleets or cause any large trouble. I fear though that this situation will only embolden them."`
			`	Just then, Alondo pokes his head into the room looking somewhat exasperated. "Captain <last>. Could you give Remington a hand? He's out on the landing pads." He disappears again without waiting for a reply, and you hear his heavy footsteps grow quieter as he walks away.`
			`	"You go help Remington," Xilin says. "I'll continue my work here."`

mission "Hai Reveal: Meet the Team: Samuel"
	name "High Tension"
	description "Remington has heard news of tensions between new arrivals and local Hai on <stopovers>. Return to <destination> when finished."
	source "Hai-home"
	stopover "Cloudfire"
	passengers 2
	blocked "You will need <capacity> in order to take this mission."
	to offer
		has "hr: meet the team"
		has "Hai Reveal: Meet the Team: Xilin: done"
	on offer

> 		log "People" "Terry Adrianopoulos" `Terry Adrianopoulos is the secretary of Samuel Remington, or at least that's how he puts it. Remington mentioned an internal political situation within the Syndicate while glancing at Terry, so there may be something larger at play with her presence. Hopefully it doesn't conflict with resolving the situation with the Hai.`

Reviewer 8: I don't feel like logbook entries should refer to events this specifically. Putting them here transforms these details from foreshadowing to heavy-handedly telling the player exactly what they need to think. They should instead be more general descriptions of characters.
```suggestion
		log "People" "Terry Adrianopoulos" `Terry Adrianopoulos is a member of the Syndicate assisting Samuel Remington in dealing with the situation regarding the Hai. Samuel claims that Terry is his secretary.`
```

		conversation
			`You exit the building onto the landing pads with Sayari following behind you and find the tall man and the woman from the front row of the briefing waiting by your ship.`
			branch met
				has "met remington"
			action

> 				log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public relations. He is acting as the Syndicate's representative in helping to deal with the situation regarding the Hai.`

Reviewer 8: Putting the internal Syndicate conflict in Sam's entries, since it's still pretty important.
```suggestion
				log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public relations. He is acting as the Syndicate's representative in helping to deal with the situation regarding the Hai, and thinks the role will be a good break from the Syndicate's current internal turmoil.`
```

			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department. You may not have heard of me before, but I'm sure you've heard of my work. I..." He stops himself. "You know what, nevermind. We're not here to make small talk. We have bigger things to worry about. I'm looking forward to working alongside you to handle this emerging crisis."`
				goto terry
			label met
			action

> 				log "People" "Samuel Remington" `Remington is being sent as the representative of the Syndicate to help deal with the situation regarding the Hai.`

Reviewer 8: ```suggestion
				log "People" "Samuel Remington" `Remington is being sent as the representative of the Syndicate to help deal with the situation regarding the Hai. He considers his new role as a good break from the Syndicate's current internal turmoil.`
```

			`	"Alondo said you needed my help," you say.`
			label help
			`	"Ah, yes. Well you see, I'd like to travel to <stopovers> to get the lay of the land around the conflict between the two Hai groups. Care to give me a ride there? It'd be just the two of us."`
			`	"I will be accompanying you," Sayari says.`
			`	"Just the three of us, then," Remington adds. "The Wah Ki system is where Mr. Campbell's video that's been plastered all over the news was taken. I imagine any new arrivals attempting to look into the validity of the video themselves would find their way there, so it'd be worth checking in on the place."`
				accept
	on stopover
		conversation
			`It doesn't take long to prove Remington's hunch correct. You find a group of new arrivals gathered by a group of Hai captains and crewmen, asking them question after question. "Why are you guys hiding up here?" "Where can I buy some of those weapons of yours?" "Do you eat acorns?" Some of the Hai look intrigued by the attention they're being given, but others look annoyed.`

> 			`	One of the human captains seems to be riling the others up and being rather hostile with his comments. "Do you really think you could invade us? You're like half my size, I could take you on in a fight any day."`

Reviewer 8: ```suggestion
			`	One of the human captains seems to be riling the others up and being rather hostile with his comments. "Do you really think you could invade us? You're like half my size; I could take you on in a fight any day."`
```

			`	Remington shakes his head. "I'm going to step in before this turns ugly," he says while pulling a small device out of his pocket. It's not anything that you recognize, but it has a Hai appearance to it.`
			`	He shouts to gain everyone's attention before tossing the device between the two groups. As it lands, a life-sized image of a Pug appears, looking almost realistic aside from the fact that its facing angle seems to follow your perspective. Most of the human captains jump back.`
			`	"Can anyone tell me what that is?" Remington asks.`
			`	"That's one of those Pug things!" says one of the human captains.`
			`	"Correct. And I take it you don't like them."`

> 			`	"Why would I like someone who invaded my home planet?" says another captain.`

Reviewer 8: I don't think that this person would refer to the Pug as "someone".
```suggestion
			`	"Why would I like something that invaded my home planet?" says another captain.`
```

			`	"Exactly. So why are you doing that to these Hai here? What's the issue you take with them, anyway?"`
			`	"Well, I'm just..." the captain begins to trip through a response before being cut off.`
			`	"Captain <last>," Remington says while turning to you. "Can you tell me what the difference between the Hai and the Pug is?"`
			choice
				`	"The Hai aren't invading us."`
				`	"The Hai are cuter."`
					goto cute
				`	"The Hai are friendly."`
					goto friendly
			`	"But they could!" one of the captains pipes up.`
			`	"Really?" Remington responds. "So because these Hai here are fighting pirates of their own species, you think they could invade us? Could the Syndicate's security forces invade the Republic because they fight off pirates, or the Republic decide to invade the Free Worlds because they fight off pirates? Maybe the answer is yes, but there's a big difference between could and will.`

> 				goto sayari

Reviewer 8: ```suggestion
				goto sayari

```

			label cute
			`	"I've seen cute dogs turn fearsome before!" one of the captains pipes up.`

> 			`	Remington gives the captain (and you) a slight look of disbelief, but he rolls with it. "Ok, so maybe cute things can be fearsome, but has a fearsome dog ever invaded your home planet?`

Reviewer 8: ```suggestion
			`	Remington gives the captain (and you) a slight look of disbelief, but he rolls with it. "Okay, so maybe cute things can be fearsome, but has a fearsome dog ever invaded your home planet?`
```

				goto sayari
			
			label friendly
			`	"But they might not always be!" one of the captains pipes up.`

> 			`	"Perhaps not," Remington responds, "but is that any justification for you being hostile now? People had the same fear of the Quarg centuries ago when they were first discovered, but have the Quarg been anything but friendly in that time?`

Reviewer 8: I think there should be a line break here?
```suggestion
			`	"Perhaps not," Remington responds, "but is that any justification for you being hostile now? People had the same fear of the Quarg centuries ago when they were first discovered, but have the Quarg been anything but friendly in that time?`

```

			`	"Ah, yes," Sayari says. "Many decades of interaction with humans has led us to accommodate much of our technology to your needs. Humans are our guests here, so we saw it only right to make you feel more comfortable. A shame to see a few bad apples may spoil the bunch."`
			`	"You have apples up here?" Remington asks.`
			`	Sayari chuckles. "Just as we have adapted our technology to human needs, we have also imported many human foods and learned human sayings. It has all been very exciting."`
			`	Sayari pauses for a moment. "Our civilization is tens of thousands of years old. There is little in that time that we have not done on our own. But seeing such a young species has led to much excitement. I wish for that excitement to not turn into strife. This is what I explained to the Hai on <planet stopovers>. 'These humans are young. Be patient with them.'"`
			choice
				`	"Glad to see that you still have a positive view of us."`

> 				`	"Well we're here to do our best to make that wish come true."`

Reviewer 8: ```suggestion
				`	"Well, we're here to do our best to make that wish come true."`
```

> 			`	Sayari goes on about the history between Hai and humanity, how a colony exploration team sanctioned by the Republic found the wormhole in Ultima Thule and made first contact between the Hai and Republic. Early on, they used the Quarg as translators between the two parties, but eventually the Hai began to learn the human language. Some humans also began to learn Hai, but it seems much more difficult for a human to speak Hai than the other way around.`

Reviewer 8: ```suggestion
			`	Sayari goes on about the history between Hai and humanity, how a colony exploration team sanctioned by the Republic found the wormhole in Ultima Thule and made first contact between the Hai and the Republic. Early on, they used the Quarg as translators between the two parties, but eventually the Hai began to learn the human language. Some humans also began to learn Hai, but it seems much more difficult for a human to speak Hai than the other way around.`
```

	stopover "Earth"
	to offer
		has "Hai Reveal: Meet the Team: Terry: done"
	on offer
		conversation
			`Terry quickly takes a copy of your ship's system logs upon your return. "Thank you, Captain. I'll have these analyzed in no time." Terry runs back into the building just as Sayari comes out carrying a small box.`
			`	"Captain <last>. I know we are asking much of you, but I have a task of my own." She hands you the box, which displays official Hai government branding. Inside is an encrypted data chip and a metallic badge, with a symbol minutely engraved into the front that must be the Hai government's emblem. Surrounding it in a circle is some sort of pattern, which reminds you of the codes found on identification cards and licenses in human space. On the other side, there is extensive writing in a script that looks to be Hai. From this, you presume that the badge is some sort of diplomatic ID.`
			`	"Please deliver this message to the Republic. I trust that they will disseminate it to your other governments. This badge should make it easier if they give you trouble."`
			choice
				`	"Consider it done."`
					accept
				`	"Various? We only have two governments."`

> 			`	"Oh," she says with a smirk, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`

Reviewer 8: Sayari doesn't say "various", though?

More generally, I feel like Sayari's response to this is a little awkward, like she's more smug that the player is wrong than honestly telling them why the Hai consider the Syndicate as a separate government.

Author: The "various" is a holdover from the previous introduction, as is Sayari's response. But I'm now realizing that this doesn't really line up with my plans for the Syndicate, so this whole but can just be tossed.

> 				`	"Various? We only have two governments."`
> 			`	"Oh," she says with a smirk, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`
> 				accept

Author: ```suggestion
```
